From 57da7b3252f9cbcf8ba536a6bde7a92937b5d7c8 Mon Sep 17 00:00:00 2001
From: Tomi Valkeinen <tomi.valkeinen@nokia.com>
Date: Fri, 26 Jun 2009 14:50:47 +0300
Subject: [PATCH 124/146] DSS2: DSI: Increase framedone timeout to 1s

It seems that with heavy CPU load it may take more time
until the work thread wakes up. I was unable to increase
the priority so that the thread would wake up earlier.
---
 drivers/video/omap2/dss/dsi.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/omap2/dss/dsi.c b/drivers/video/omap2/dss/dsi.c
index f3553f8..97a67e5 100644
--- a/drivers/video/omap2/dss/dsi.c
+++ b/drivers/video/omap2/dss/dsi.c
@@ -2785,7 +2785,7 @@ static int dsi_update_thread(void *data)
 			dsi_update_screen_dispc(device, x, y, w, h);
 
 			/* wait for framedone */
-			timeout = msecs_to_jiffies(500);
+			timeout = msecs_to_jiffies(1000);
 			timeout = wait_event_timeout(dsi.waitqueue,
 					dsi.framedone_received == true,
 					timeout);
-- 
1.6.2.4

